### ISSUE TITLE
"phobetor_rabbitmq_supervisor" service is private

### ISSUE THREAD

**Issue by coudenysj** [Phobetor/rabbitmq-supervisor-bundle#30]
In the commands, the "phobetor_rabbitmq_supervisor" is fetched from the container, but it is marked as private in de container.

```
User Deprecated: The "phobetor_rabbitmq_supervisor" service is private, getting it from the container is deprecated since Symfony 3.2 and will fail in 4.0. You should either make the service public, or stop using the container directly and use dependency injection instead
```

Should we mark the service as public? Or inject them in the commands?

https://github.com/Phobetor/rabbitmq-supervisor-bundle/search?utf8=%E2%9C%93&q=%22get%28%27phobetor_rabbitmq_supervisor%27%29%22&type=

**Comment by gregurco** [comment]
@coudenysj it was fixed in this PR #35: https://github.com/Phobetor/rabbitmq-supervisor-bundle/pull/35/files#diff-a08c782a2ee112dc4ee0a0152ba6c6ccR8

**Comment by Phobetor** [comment]
Should be handled by #35 which is merged to master and released as new version [3.0.0](https://github.com/Phobetor/rabbitmq-supervisor-bundle/releases/tag/3.0.0).
